sdk/eventhub/event-hubs/package.json

```diff
@@ -29,18 +29,18 @@
     "dist/",
     "dist-esm/src/",
     "src/",
-    "typings/src",
+    "types/src",
     "tsconfig.json"
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1 && npm run extract-api",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
     "build:samples": "cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1",
+    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1 && npm run extract-api",
+    "build": "tsc -p . && rollup -c 2>&1 && npm run extract-api",
```

Comment:
What is the reason to include the extraction for test output?

Reply:
I could go either way on it. Basically, I was thinking of testing as a way of saying "ensure that what I have made would pass CI". As part of that, we'd want to run the api extractor to flag any possible errors, too.

Reply:
Since one would need to run the `build` script before running the tests, wouldnt adding the extract step for just the `build` script be enough?